pkg/handlers/generic/lifecycle/csi/nutanix-csi/handler.go

```diff
@@ -24,14 +23,31 @@ import (
 )
 
 const (
-	defaultHelmRepositoryURL              = "https://nutanix.github.io/helm/"
-	defaultStorageHelmChartVersion        = "v2.6.6"
-	defaultStorageHelmChartName           = "nutanix-csi-storage"
-	defaultStorageHelmReleaseNameTemplate = "nutanix-csi-storage-%s"
-
-	defaultSnapshotHelmChartVersion        = "v6.3.2"
-	defaultSnapshotHelmChartName           = "nutanix-csi-snapshot"
-	defaultSnapshotHelmReleaseNameTemplate = "nutanix-csi-snapshot-%s"
+	defaultHelmRepositoryURL           = "https://nutanix.github.io/helm/"
+	defaultStorageHelmChartVersion     = "v2.6.6"
+	defaultStorageHelmChartName        = "nutanix-csi-storage"
+	defaultStorageHelmReleaseName      = "nutanix-csi-storage"
+	defaultStorageHelmReleaseNamespace = "ntnx-system"
+
+	defaultSnapshotHelmChartVersion     = "v6.3.2"
+	defaultSnapshotHelmChartName        = "nutanix-csi-snapshot"
+	defaultSnapshotHelmReleaseName      = "nutanix-csi-snapshot"
+	defaultSnapshotHelmReleaseNamespace = "ntnx-system"
+
+	defaultCredentialsSecretName = "nutanix-csi-credentials"
+)
+
+var (
+	defaultStorageClassParameters = map[string]string{
```

Comment:
These defaults needed vars from this package so moved it here.
But also moved the AWS defaults to the AWS file, the utils package should be more generic.